website/content/docs/plugins/plugin-portal.mdx

```diff
@@ -121,7 +121,7 @@ stability guarantees on these plugins are established by their authors.
 
 Plugin authors who wish to have their plugins listed may file a submission via a
 [GitHub issue][github-issue] or directly open a pull request with changes to
-[this page][plugin-portal-mdx].
+[this page](https://github.com/hashicorp/vault/blob/main/website/content/docs/plugins/plugin-portal.mdx).
```

Comment:
FWIW, I'd just update the link below (line 143), not here.

Reply:
You are right, didn't notice the anchor. Changed, thanks.